test/backend/variables.py

```diff
@@ -164,6 +164,14 @@ def get_args_from_env():
         default=os.getenv("TEST_OPTLEVEL", "3"),
         help="set compiler optimization level (default: 3 if TEST_OPTLEVEL env var not set)",
     )
+    parser.add_argument(
+        "-t",
+        "--type",
+        type=str,
+        choices=["node", "model"],
+        default=os.getenv("TEST_TYPE", ""),
```

Comment:
Is empty string a choice?

Reply:
I see that all test cases are selected if type is not defined.

Reply:
Yes. `if not args.type` will be true if the option is an empty string and all tests will be enabled by default.